### DISCUSSION TITLE
How to make widget card on equal height without overriding the view?

### DISCUSSION THREAD

**Discussion by hazwan91** [filamentphp/filament#14768] [Help]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

I'm having a problem to make my widget card on same height especially for chart. Different screen size, different height it becomes.

i try to set maxheight still not working..

![Screenshot 2024-11-12 082751](https://github.com/user-attachments/assets/8caab9e1-6cf5-45ef-8c31-5b20b017437c)

I try to find something like extraAttributes to add extra classes... but i can't find one and don't know where to add it.

I look at the filament demo, it seems resize in proper manner.. how to do that?

Please help

**Comment by leandrocfe** [comment]
[maxHeight](https://filamentphp.com/docs/3.x/widgets/charts#setting-a-maximum-chart-height) should work. What code are you using?

**Reply by hazwan91** [reply]
i use this `protected static ?string $maxHeight = '450px';` i set the same value, but both height not the same.

but seems not working like wanted to.. both chart can't keep up with each other's height..
i even try to use `%`, `vh` and so on.. can't match to each other. 

i plan to set some custom styling but dunno where to put the extra styling..

I am not sure if `chart.js` is related to this.

**Reply by leandrocfe** [reply]
hum.. not sure about the chart.js, but I think you can do this using [apexcharts](https://filamentphp.com/plugins/leandrocfe-apex-charts), because you can also set the chart height in the options..

**Reply by leandrocfe** [reply]
[take a look at this example](https://github.com/leandrocfe/filament-demo/blob/apex-charts-plugin-v3/app/Filament/Widgets/CustomersChart.php)

```php
return [
    'chart' => [
        'type' => 'line',
        'height' => 250 ...
    ],
```

**Comment by hazwan91** [comment]
Just for everyone information. Its working fine and maintain equal height if i set i to lower height for example `200px` . its just that, the chart becomes small on every screen size.

if i set it higher than that, the height will be different.. It's only looking equal in higher screen size as well but not in smaller screen such as `md` or `sm` or even `lg`. I don't mind in mobile screen since i make it 12 grid by default.

maybe need to have a method for minHeight? 

I guess lowering the maxHeight should be fine for now.. I try to look for other `simple` solution.
